composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
I'm sure they have good reasons for this, but we're in a different boat here. Why would we drop support for something that works? I'm not against dropping support for ancient stuff, but it's important not to go overboard here.